skunk.gemspec

```diff
@@ -52,6 +52,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rubocop", "~> 1.0"
   spec.add_development_dependency "simplecov", "~> 0.18"
   spec.add_development_dependency "simplecov-console", "0.5.0"
-  spec.add_development_dependency "vcr", "~> 6.0.0"
```

Comment:
Use RUBY_VERSION in a `gemspec` file raised this cop.
https://github.com/rubocop/ruby-style-guide/issues/763